corehq/apps/translations/models.py

```diff
@@ -113,8 +135,9 @@ class TransifexBlacklist(models.Model):
             ('list', 'Case List'),
         )
     )
-    field_name = models.CharField(max_length=255)
-    display_text = models.CharField(max_length=255)
+    field_name = models.TextField(help_text=FIELD_NAME_HELP)
+    display_text = models.TextField(
+        help_text="The default English translation for this detail/list")
```

Comment:
this is a bit risky. In case the source lang is changed in the app, it would be skipped. 
One way to avoid that would be to ensure that all blacklisted translations are present at least once? Can't think of a better way for this but definitely worth considering.

Also should we make this optional so that one needs to enter it only why needed to differentiate between similar case list/details?

Reply:
>  In case the source lang is changed in the app,

That won't be true for ICDS though right? I'm imagining that ICDS is the only one that would be using this  while REACH is being developed

Reply:
we did change the source language from hindi to english on icds. But a couple of projects are looking at using transifex integration and hence might use this as well.

Reply:
Thanks I didn't realize that, or that other projects would be using transifex (since this used to live in the custom/icds folder). I also didn't see that we use default_lang_index a couple of lines above this, so thanks for pointing that out as well